client/asset/interface.go

```diff
@@ -335,6 +335,9 @@ type RedeemForm struct {
 	// suggestion in any way, but obviously fees that are too low may result in
 	// the redemption getting stuck in mempool.
 	FeeSuggestion uint64
+	// AssetVersion is the swap protocol version, which may indicate a specific
+	// contract or form of contract.
+	AssetVersion uint32
```

Comment:
This is good for now. Can rethink in or after #1320. 

Side note: It may be possible we don't need the `FromVersion` and `ToVersion` added to the `OrderMetaData` in #1301 after all.

Reply:
Pretty sure I'll clean it out in 1320, merging with this solution now